### PR TITLE
Redefine purpose of the project in a more concrete way

### PR DESCRIPTION
**Pull request by dev-ng**
Old: making a breakthrough - judgment category in this context, it is the result that the community can give on the outcome of the work. It can't be set as a goal or purpose.

Old: advance computing - too wide definition.

### CODE REVIEW

**Comment by sergefdrv** [comment]
> making a breakthrough - judgment category in this context, it is the result that the community can give on the outcome of the work. It can't be set as a goal or purpose

Making a breakthrough is not the purpose -- it's the spirit.

> advance computing - too wide definition

I believe it's just right for the purpose, given that it's followed by "making highly reliable distributed systems more practical to implement". Note that it's the purpose, not the goal.

**Comment by sergefdrv** [comment]
Though, as a non-native English speaker, I may have confused the nuances in the meaning of the word "purpose". Maybe "mission" would fit better?

**Comment by dev-ng** [comment]
> Making a breakthrough is not the purpose -- it's the spirit.

Okay, but then it should be used like that. Now we have:

"XYZ is about" **spirit** "in this for that."

I'll give you a specially exaggerated example:

"Github is about making an excitement in ... for ...!"

> > advance computing - too wide definition

The thing is that the current description lacks a clear practical explanation without pathos and too wide definitions. It should give clear understanding and directions of what needs to be done. You can find all the needed pieces spread all over the texts on different pages, but not all together in one place.

Developers should be able to realize if they can find here what they are looking for or understand if they are at the right place where they could utilize their skills by contribution.

Managers should be able to see if the project could help them with their business challenges.

I don't think they are interested in "making a breakthrough" or "advancing computing".

**Comment by sergefdrv** [comment]
> > Making a breakthrough is not the purpose -- it's the spirit.
> 
> Okay, but then it should be used like that. Now we have:
> 
> "XYZ is about" **spirit** "in this for that."
> 
> I'll give you a specially exaggerated example:
> 
> "Github is about making an excitement in ... for ...!"

So what's wrong with it?


> The thing is that the current description lacks a clear practical explanation without pathos and too wide definitions. It should give clear understanding and directions of what needs to be done. 

The description on the main page is one paragraph condensing the content of the About page, which describes the project as it is right now. It doesn't attempt to commit to anything that is not yet certain at this point or prematurely restrict the future of the project; it will get more concrete over time.

The project is quite ambitious (set on making a breakthrough), so some pathos is appropriate here.

> You can find all the needed pieces spread all over the texts on different pages, but not all together in one place.

What is particularly missing on the [About](https://replica-io.dev/about) page that one can already find in other places?

> Developers should be able to realize if they can find here what they are looking for or understand if they are at the right place where they could utilize their skills by contribution.

The project is currently in initial research stage, which one can see from the [Roadmap](https://replica-io.dev/about#-roadmap): M0.1 is not marked as complete. The wide definitions suggest that one can contribute by sharing and discussing high-level ideas.

> Managers should be able to see if the project could help them with their business challenges.

I don't think that the project at this stage would be anyway interesting for managers, though they could contribute by telling what business challenges this kind of project could solve for them. Again, it will get focused on concrete problems depending on concrete discoveries made during the initial research stage.

> I don't think they are interested in "making a breakthrough" or "advancing computing".

It's probably not the right place for those who are not interested in doing hat.

File: src/pages/index.jsx [hunk 1]
```diff
@@ -56,15 +56,16 @@ export default function Home() {
             <p className="row">
               <p className="col col--8 col--offset-2">
                 <p>
-                  Replica_IO is about making a breakthrough in designing
-                  and implementing distributed protocols for the future of
-                  decentralized computing! 🚀
+                  Replica_IO is about consolidation of the best available
+                  ideas and principles to-date in designing and
+                  implementing distributed protocols for the future of
+                  decentralized computing.
```
Comment (sergefdrv) [inline src/pages/index.jsx:62]: This project is really about making a step forward, not just trying to fit some disparate ideas together, though learning from them should be helpful.

Comment (sergefdrv) [inline src/pages/index.jsx:62]: This project is really about making a step forward, not just trying to fit some disparate ideas together, though learning from them should be helpful.

**Pull request closed without merging**